<launch>
<!--
- LT(6): Brake
- RT(7): Throttle
- left(0): steering 
- A(0): v-1 gear (set the feasible velocity interval into [0, 0.5])
- B(1): v-2 gear (set the feasible velocity interval into [0.3, 0.8])
- X(2): w-1 gear (set the feasible angular velocity interval into [-0.5, 0.5])
- Y(3): w-2 gear (set the feasible angular velocity interval into [-1, 1])

驾驶经验总结：
方案1： 定速行驶 （speed=23）
  - 赛道1-3: steer=3,  弯道线处开始转向，1.0左右，控制距弯道线距离
  - 赛道4:   提前拐，不要快撞上了才拐
                       或者中线拐入，10左右能更好拐入，别拐早了；
  - 赛道5:   steer=15, 注意使用小角度！4左右
  - 赛道6-8: steer=3, 换挡->0.4左右，注意连续控制

 -->
<!-- Start the joystick driver to publish the /joy topic -->
<node pkg="joy" type="joy_node" name="joystick"/>

<!-- Start the telop node to control the car-->
<node pkg="teleop_robot" type="teleop_joy_uisee_steer.py" name="teleop_uisee" output="screen">
  <param name="v_delta" value="10.0"/>
  <param name="v_c" value="20.0" />
  <param name="s_turn" value="2.0"/>
  <param name="v_min" value="10.0"/>
  <param name="v_max" value="25.0"/>
  <param name="gear_w1" value="5.0"/>
  <param name="gear_w2" value="10.0"/>
  <param name="gear_w3" value="8.0"/>
  <param name="target_ip" value="192.168.1.108"/>
  <param name="data_path" value="/home/ubuntu16/Datasets/UISEE_SIM/I"/>
  <param name="num_frames" value="11" />
  <param name="img_count" value="10388" />


  
</node>


</launch>